dataactvalidator/filestreaming/csvS3Reader.py

```diff
@@ -24,18 +18,23 @@ def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
         s3Bucket = s3connection.lookup(bucket)
         self.s3File = s3Bucket.lookup(filename)
         self.isLocal = False
-        if(self.s3File == None):
-            raise ValueError("".join(["Filename provided not found on S3: ",str(filename)]))
+        if not self.s3File:
```

Comment:
Very minor, but recommended practice for testing against `None` (as opposed to `False`, the empty string, etc) is:
```python
if self.s3File is None:
```